recipes/renode-cli/build.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+# Consolidated script logic
+main() {
+    export PATH="${DOTNET_ROOT}/dotnet:${PATH}"
+
+    dotnet_version=$(dotnet --version)
+    framework_version=${dotnet_version%.*}
+
+    # Patch project files (combined find and sed)
+    find lib src tests -name "*.csproj" -exec sed -i -E "s/([>;])net6.0([<;])/\1net${framework_version}\2/g" {} \;
+
+    # Remove obj and bin directories (combined find)
+    find . -name "obj" -o -name "bin" -type d -exec rm -rf {} +
+
+    # Fix typo in Renode_NET.sln
+    sed -i -E 's/(ReleaseHeadless\|Any .+ = )Debug/\1Release/' Renode_NET.sln
+
+    # Prepare for build (using more robust path)
+    mkdir -p "${SRC_DIR}/src/Infrastructure/src/Emulator/Cores/bin/Release/lib"
+    ln -sf ${PREFIX}/lib/renode-cores/* "${SRC_DIR}/src/Infrastructure/src/Emulator/Cores/bin/Release/lib"
+
+    rm -f "${SRC_DIR}/src/Infrastructure/src/Emulator/Cores/translate*.cproj"
+
+    # Build with dotnet (combined commands and simplified logic)
+    mkdir -p "${SRC_DIR}/output/bin/Release/net${framework_version}"
+    cp "${SRC_DIR}/src/Infrastructure/src/Emulator/Cores/${target_platform%%-*}-properties_NET.csproj" "${SRC_DIR}/output/properties.csproj"
+
+    dotnet build \
+      -p:GUI_DISABLED=true \
+      -p:Configuration=ReleaseHeadless \
+      -p:GenerateFullPaths=true \
+      -p:Platform="Any CPU" \
+      ${SRC_DIR}/Renode_NET.sln
+    echo -n "dotnet" > "${SRC_DIR}/output/bin/Release/build_type"
+
+    # Copy LLVM library (simplified logic)
+    LLVM_LIB="libllvm-disas"
+    cp "lib/resources/llvm/$LLVM_LIB$SHLIB_EXT" "${SRC_DIR}/output/bin/Release/libllvm-disas$SHLIB_EXT"
+
+    # Install procedure (combined mkdir)
+    mkdir -p ${PREFIX}/bin ${PREFIX}/libexec/${PKG_NAME} ${PREFIX}/share/${PKG_NAME}/{scripts,platforms,tests}
+
+    cp -r ${SRC_DIR}/output/bin/Release/net${framework_version}/* ${PREFIX}/libexec/${PKG_NAME}/
+    cp -r ${SRC_DIR}/scripts/* "${PREFIX}/share/${PKG_NAME}/scripts/"
+    cp -r ${SRC_DIR}/platforms/* "${PREFIX}/share/${PKG_NAME}/platforms/"
+
+    # Copy licenses (simplified conditional)
+    mkdir -p ${SRC_DIR}/license-files
+    if [[ "${target_platform}" == "osx-*" ]]; then
+        tools/packaging/common_copy_licenses.sh ${SRC_DIR}/license-files macos
+    else
+        tools/packaging/common_copy_licenses.sh ${SRC_DIR}/license-files linux
+    fi
+
+    # Create renode script (using heredoc)
+    cat > "${PREFIX}/bin/renode" <<EOF
+#!/bin/sh
+exec "\${DOTNET_ROOT}"/dotnet exec "\${CONDA_PREFIX}"/libexec/renode-cli/Renode.dll "\$@"
+EOF
+    chmod +x "${PREFIX}/bin/renode"
+
```

Comment:
We still need to call `dotnet-licenses` here to get all the third party license used in the actual .NET DLLs.  The licenses copied by that helper script don't actually cover all of those.   I think you had this in a previous version of the script?

Reply:
this is sufficient:
`    dotnet-project-licenses --input "$SRC_DIR/src/Renode/Renode_NET.csproj" -d "$SRC_DIR/license-files" -f "txt"
`?